bodhi/server/models/models.py

```diff
@@ -1709,39 +1709,43 @@ def check_requirements(self, session, settings):
         return True, "All checks pass."
 
     def check_karma_thresholds(self, db, agent):
-        """Check if we have reached either karma threshold, and call set_request if necessary"""
-        if not self.locked:
-            if self.status is UpdateStatus.testing:
-                # If critical update receives negative karma disable autopush
-                if self.critpath and self.autokarma and self._composite_karma[1] != 0:
-                    log.info("Disabling Auto Push since the critical update has negative karma")
-                    self.autokarma = False
-                elif self.stable_karma not in (0, None) and self.karma >= self.stable_karma:
-                    if self.autokarma:
-                        log.info("Automatically marking %s as stable" % self.title)
-                        self.set_request(db, UpdateRequest.stable, agent)
-                        self.request = UpdateRequest.stable
-                        self.date_pushed = None
-                        notifications.publish(
-                            topic='update.karma.threshold.reach',
-                            msg=dict(update=self, status='stable'))
-                    else:
-                        # Add the 'testing_approval_msg_based_on_karma' message now
-                        log.info((
-                            "%s update has reached the stable karma threshold and can be pushed to "
-                            "stable now if the maintainer wishes") % self.title)
-                elif self.unstable_karma not in (0, None) and self.karma <= self.unstable_karma:
-                    log.info("Automatically unpushing %s" % self.title)
-                    self.obsolete(db)
-                    notifications.publish(
-                        topic='update.karma.threshold.reach',
-                        msg=dict(update=self, status='unstable'))
-            else:
-                # Ignore karma thresholds for pending/stable/obsolete updates
-                pass
-        else:
+        """
+        Check if we have reached either karma threshold, and
+        call set_request if necessary
```

Comment:
The 80 column line length is the one PEP-8 provision that I actively oppose because it was based on computer monitors that are so old these days that there's no excuse for coding on them (they are bad for your eyes!). I personally prefer to limit line lengths to 100 columns. Thus, I am actually fine with the wrapping that this docblock was at previously. However, I don't mind if you really want this change.